src/chord.js

```diff
@@ -79,32 +79,28 @@ const modeMap = {
 };
 
 // Alternate names for chords
-modeMap.Maj = modeMap.maj;
```

Comment:
Yes, the user can input anything they want, like CMaj7 or Dim7, and it will be transformed into lower case to be compared to the object of chords.